lib/components/user/monitored-trip/trip-notifications-pane.js

```diff
@@ -44,64 +103,66 @@ class TripNotificationsPane extends Component {
     } else {
       notificationSettingsContent = (
         <>
-          <FormGroup>
-            <ControlLabel id='is-active-label'>Would you like to receive notifications about this trip?</ControlLabel>
-            <div role='group' aria-labelledby='is-active-label'>
-              <Radio
-                checked={monitoredTrip.isActive}
-                name='isActive'
-                onChange={this._handleIsActiveChange}
-                value
+          <h4>Notify me via {notificationChannelLabels[notificationChannel]} when:</h4>
+          <SettingsListWithAlign>
+            <li>
+              <Select
+                label='There is a realtime alert flagged on my journey'
+                name='notifyOnAlert'
               >
-              Yes
-              </Radio>
-              <Radio
-                checked={!monitoredTrip.isActive}
-                name='isActive'
-                onChange={this._handleIsActiveChange}
-                value={false}
+                <option value>Yes (default)</option>
+                <option value={false}>No</option>
+              </Select>
+            </li>
+            <li>
+              <Select
+                label='An alternative route or transfer point is recommended'
+                name='notifyOnItineraryChange'
               >
-              No
-              </Radio>
-            </div>
-            <HelpBlock>
-            Note: you will be notified by {notificationChannelText}. This can be changed in
-            your account settings once the trip has been saved.
-            </HelpBlock>
-          </FormGroup>
-
-          <FormGroup>
-            <ControlLabel>When would you like to receive notifications about delays or disruptions to your trip?</ControlLabel>
-
-            <FormGroup>
-              <ControlLabel>Check for delays or disruptions:</ControlLabel>
-              <Field
-                as={FormControl}
-                componentClass='select'
-                name='leadTimeInMinutes'
+                <option value>Yes (default)</option>
+                <option value={false}>No</option>
+              </Select>
+            </li>
+            <li>
+              <Select
+                label='There are delays or disruptions of more than'
+                name='departureVarianceMinutesThreshold'
```

Comment:
I'm just noticing there is no field for `arrivalVarianceMinutesThreshold`. Should there be?

Reply:
@evansiroky In https://github.com/opentripplanner/otp-react-redux/pull/323/commits/b30a2ef5c4c3452aa88ab16eab466eb6ecbc4a0a I split the target threshold field depending on whether the trip is arriveBy.

Reply:
Hmm... I'm somewhat of the mind that we should just set both thresholds to the same value (chosen by the single select component here) regardless of whether the trip is arriveBy/departAt. If I had a trip that was set to arrive by 9am. I would still really like to know if the first leg is running early by 15 minutes (an extreme case, but still feasible). Thoughts?

Reply:
I am in favor of using a single selector, so @landonreed's idea may be better overall?

Reply:
Per team discussion, I implemented single UI field that sets the value for both the departure and arrival delay thresholds in https://github.com/opentripplanner/otp-react-redux/pull/323/commits/2dd18214428e945fb4ea19511ab205f00233acf7.